lib/MetaCPAN/Script/Release.pm

```diff
@@ -292,6 +307,7 @@ sub import_tarball {
                 = -d $File::Find::name
                 ? dir($File::Find::name)
                 : file($File::Find::name);
+            return if $self->_is_broken_file($File::Find::name);
```

Comment:
Since you're not using the `$child` var that gets initialized above, you could move this return to the top
to avoid attempting to call `file()` on the bad value.

Because of the infrequency of these files it won't really be much of a performance gain, but it makes more sense than having it after.